tools/shared/pragma.c

```diff
@@ -430,6 +431,7 @@ struct c {
 static struct c table[] = {
     {"align", SW_ALIGN, false, S_NONE, S_NONE},
     {"altcode", SW_ALTCODE, true, S_LOOP, S_LOOP | S_ROUTINE | S_GLOBAL},
+    {"forceinline", SW_FORCEINLINE, true, S_ROUTINE, S_ROUTINE | S_GLOBAL},
```

Comment:
Quoting myself from the old pull request:

> I don't think you need to support noforceinline since another pragma named noinline exists for exactly the same purpose. Also, your test cases did not cover the use of noforceinline.

What I meant to suggest was that you could change the `true` to `false` on this line.

Reply:
The `noforceinline` test would be unnecessary.

Reply:
Oh apologies, I missed that.
Updated the code.